api_app/ingestors_manager/ingestors/threatfox.py

```diff
@@ -12,14 +12,11 @@
 
 
 class ThreatFox(Ingestor):
+    url: str
     days: int
 
-    BASE_URL = "https://threatfox-api.abuse.ch/api/v1/"
-
     def run(self) -> Iterable[Any]:
-        result = requests.post(
-            self.BASE_URL, json={"query": "get_iocs", "days": self.days}
-        )
+        result = requests.post(self.url, json={"query": "get_iocs", "days": self.days})
```

Comment:
Why did you change the URL? Why someone has to change the URL to something differently?

Reply:
I changed the `BASE_URL` to `url` in order to enable the healthcheck feature (it's present, why disable it completely?).
I made it to do things the same way, as if it were a standard. If it doesn't go well I'll put it back as before.